src/NHibernate/Hql/Ast/ANTLR/Tree/FromElement.cs

```diff
@@ -482,6 +482,17 @@ public IType GetPropertyType(string propertyName, string propertyPath)
 		}
 
 		public virtual string GetIdentityColumn()
+		{
+			var cols = GetIdentityColumns();
+			string result = StringHelper.Join(", ", cols);
```

Comment:
Any reason using `StringHelper.Join` instead of `string.Join`?

Reply:
Ok, got it now, just a method split.

Reply:
Yep. It was there. I didn't pay attention. I think it can be replaced with `string.Join`...